applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing_matrix_free.h

```diff
@@ -114,13 +109,22 @@ class MapperVertexMorphingMatrixFree
     ///@name Operations
     ///@{
 
+    // --------------------------------------------------------------------------
+    void InitializeMapping()
+    {
+        CreateListOfNodesOfDesignSurface();
+        CreateFilterFunction();
+        InitializeMappingVariables();
+        AssignMappingIds();
+    }
+
     // --------------------------------------------------------------------------
     void MapToDesignSpace( const Variable<array_3d> &rNodalVariable, const Variable<array_3d> &rNodalVariableInDesignSpace )
     {
         BuiltinTimer mapping_time;
         std::cout << "\n> Starting to map " << rNodalVariable.Name() << " to design space..." << std::endl;
```

Comment:
better to use the logger for prints

Reply:
I would want to introduce the logger separately to not blow up this PR any further.